scl/pgsql/pgsql.conf

```diff
@@ -35,37 +35,111 @@
 # avg read rate: 0.000 MB/s, avg write rate: 114.609 MB/s
 # system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s",,,,,,,,,""
 
-block parser postgresql-csvlog-parser(prefix(".pgsql.")) {
-    csv-parser(
-        columns(
-            "timestamp",
-            "username",
-            "database",
-            "pid",
-            "connection_from",
-            "session_id",
-            "session_line_num",
-            "command_tag",
-            "session_start_time",
-            "virtual_transaction_id",
-            "transaction_id",
-            "severity",
-            "sql_state_code",
-            "message",
-            "detail",
-            "hint",
-            "internal_query",
-            "internal_query_pos",
-            "context",
-            "query",
-            "query_pos",
-            "location",
-            "application_name",
-        )
-        delimiters(",")
-	dialect(escape-double-char)
-        flags(strip-whitespace)
-        prefix(`prefix`)
-        quote-pairs('""')
-    );
+block parser postgresql-csvlog-parser(internal(yes) prefix(".pgsql.")) {
+    channel {
+        parser { csv-parser(
+            internal(yes)
+            columns(
+                "timestamp",
+                "username",
+                "database",
+                "pid",
+                "connection_from",
+                "session_id",
+                "session_line_num",
+                "command_tag",
+                "session_start_time",
+                "virtual_transaction_id",
+                "transaction_id",
+                "severity",
+                "sql_state_code",
+                "message",
+                "detail",
+                "hint",
+                "internal_query",
+                "internal_query_pos",
+                "context",
+                "query",
+                "query_pos",
+                "location",
+                "application_name",
+                # Available from vesion 13+
+                "backend_type",
+                # Available from vesion 14+
+                "leader_pid",
+                "query_id",
```

Comment:
Can you do tests with 13+ and 14+ example logs, so we can see these fields working?